src/pybind/mgr/dashboard/run-frontend-e2e-tests.sh

```diff
@@ -10,6 +10,10 @@ stop() {
     exit $1
 }
 
+# Check if package jq is already installed. RHEL only
+packages=(jq); # Here an array variable, we can add more packages like packages=(jq docker chrome) etc
+if ! rpm -q "${packages[0]}"; then yum install -y "${packages[0]}"; fi
```

Comment:
I think it's ok to check for dependencies here before failing with a `command not found` error, but I don't think dependency management should be a concern for this script. Specially when this script is only covering RPM-based installations, while Ceph `install-deps.sh` already installs `jq` for the following distros:

- install-deps.sh (FreeBSD)
- ceph.spec.in (RPM-based distros)
- debian/control (DEB-based distros)
- alpine/APKBUILD.in

Reply:
Thank Ernesto. These points are very valid indeed. I totally missed the ceph.spec.in file. I signed my commit. Hope this is well.

Reply:
@valentinbajrami please make sure that the signed-off-part needs is included in your commit message (so you need to adapt your original commit message and force-push it into your fork/branch).
Just leave a blank line after your actual commit message and then add the following below:
``Signed-off-by: Valentin Bajrami <valentin.bajrami@target-holding.nl>``